jaxrs-clients/src/test/java/com/palantir/remoting3/jaxrs/feignimpl/GuavaOptionalAwareDecoderTest.java

```diff
@@ -132,9 +132,9 @@ public void testComplexType() {
                         new GuavaOptionalComplexType(
                                 Optional.absent(),
                                 Optional.absent(),
-                                Paths.get("bar"))),
+                                Paths.get("/bar"))),
                 Optional.of("baz"),
-                Paths.get("foo"));
+                Paths.get("/foo"));
```

Comment:
why this change?

Reply:
This is explained in https://github.com/FasterXML/jackson-databind/issues/1264 . Unfortunately previously java.nio.file.Path didn't round trip correctly. In order to fix it they changed the serialization format of Path to always write an absolute path. If you pass a relative path and serialize it will make it absolute which is hard to handle in tests

Reply:
ok